### ISSUE TITLE
What changes need to be made in next release version V2.1.2?

### ISSUE THREAD

**Issue by kanzhu-cisco** [cisco/openh264#3435]
We have some PRs need to merge to master, and release to next version. But it is blocked by travis transfer, as there is no auto-build for new pull request.

1. Build binary for apple M1,              #3423 
2. Fix mips32 build,                             #3384 
3. meson build optimize,                    #3431 
4. Fix regression caused by b527868, #3419 
5. Update ppc64el info in README,   #3425  
6. eliminate meson warnings,             #3351

**Comment by joakim-tjernlund** [comment]
What about https://github.com/cisco/openh264/pull/3382 ?
Should probably be included in next release too.

**Comment by joakim-tjernlund** [comment]
Any progress? Current release is old and buggy so a new release is needed.

**Comment by eli-schwartz** [comment]
It would be great to eliminate meson warnings by merging #3351

**Comment by joakim-tjernlund** [comment]
Will we have a release this year?
